### DISCUSSION TITLE
Base Planetary Gears : bad ...

### DISCUSSION THREAD

**Discussion by Curieux31** [almtzr/Pedro#8] [Q&A]
![1733843537045](https://github.com/user-attachments/assets/e6b35e46-2515-4e71-9182-22fa68f58296)
Hello,
Thanks for this work!!!
On the other hand, I have a problem with parts No. 4 and 16. Indeed, gears 16 have larger teeth than those of part 4....
Is there a change of scale to be expected?
Any idea?
Thanks.

**Comment by almtzr** [comment]
Hi! I’ve upgraded the planetary gear system and uploaded some parts. Please check out this commit: https://github.com/almtzr/Pedro/commit/670b64d0414ebfccba635de1956297bbeabf13a5. 

Let me know if you encounter any issues.

Please note that some parts, like the gripper, are still a work in progress.

Best regards,

**Comment by almtzr** [comment]
The STL files have been updated in accordance with the Kickstarter campaign.
